presto-main/src/test/java/com/facebook/presto/operator/TestDictionaryLookupJoinPageBuilder.java

```diff
@@ -46,7 +46,7 @@ public void testPageBuilder()
         Block block = blockBuilder.build();
         Page page = new Page(block, block);
 
-        SimpleJoinProbeFactory simpleJoinProbeFactory = new SimpleJoinProbeFactory(ImmutableList.of(BIGINT, BIGINT), new int[]{0, 1}, ImmutableList.of(0, 1), OptionalInt.empty());
+        JoinProbeFactory simpleJoinProbeFactory = new JoinProbeFactory(ImmutableList.of(BIGINT, BIGINT), new int[]{0, 1}, ImmutableList.of(0, 1), OptionalInt.empty());
```

Comment:
As there is now only one `JoinProbeFactory`, the variable should be renamed too.

(here & elsewhere)